src/test/modelconfig_test.cpp

```diff
@@ -274,6 +274,44 @@ TEST(ModelConfig, mappingOutputs) {
     EXPECT_EQ(empty, "");
 }
 
+TEST(ModelConfig, mappingRealInputs) {
```

Comment:
There are missing unit tests for model loading. It would be great to have higher level test that checks whether model loads correctly with mapping and shape/layout set with mapped name.

Reply:
Possibly also request validation to add such end to end test?

Reply:
Agree. After this MR we should have following tests (possibly some of them are already written):
-> model with mapping, trying to set shape/layout with old name -> expect FAIL
-> model with mapping, trying to set shape/layout with new mapped name -> expect PASS
-> some DAG with model mapping, using original name load should FAIL
-> some DAG with model mapping, using mapped name load should PASS